lib/assets/stylesheets/modules/_technical-documentation.scss

```diff
@@ -125,6 +125,15 @@
     border: 1px solid $code-02;
   }
 
+  pre.code-block {
+    border: $govuk-focus-width solid transparent;
```

Comment:
This is overriding the border on `pre` (line 125), which means code blocks no longer have a border:

## Before

![Screenshot 2021-03-11 at 15 18 30](https://user-images.githubusercontent.com/121939/110809742-192f8680-827d-11eb-9e5a-3f296f585044.png)

## After

![Screenshot 2021-03-11 at 15 18 24](https://user-images.githubusercontent.com/121939/110809764-1cc30d80-827d-11eb-85cc-59e7ff33c2d9.png)

We use the transparent border in the Design System so that the box 'shrink' when focused. This works there because we don't have a (visible) border on code blocks.

An alternative that doesn't change the design might be to change the padding when focused – reducing the padding from 15px to 13px to compensate for the extra 2px on the border.

Reply:
Good spot, thanks! My backend developer eyes can't see things like missing borders 🙈 

I've implemented your suggestion of dropping two pixels of padding in this force push: https://github.com/alphagov/tech-docs-gem/compare/67994d8b1e406b54e10587c57b228e359d1b5f22..0f5303b52c286d2aa3d5e51f65f9e651a5672fab

![image](https://user-images.githubusercontent.com/1696784/110814303-3a927180-8281-11eb-9102-6d7e262cbc94.png)

![image](https://user-images.githubusercontent.com/1696784/110814169-1cc50c80-8281-11eb-9196-b1423fa4cf51.png)